helm/bloom-vault/bloom-vault-env-secret-example.yaml

```
@@ -0,0 +1,24 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  annotations:
+  name: bloom-vault-env-secret
+type: Opaque
+stringData:
+  NODE_ENV: "production"
+  PIPELINE_STAGE: "staging"
```

I see this is an example, but this should likely be `"production"`.

```suggestion
  PIPELINE_STAGE: "production"
```